TeamCode/src/main/java/org/firstinspires/ftc/teamcode/modules/drive/XDrive.java

```diff
@@ -11,62 +11,64 @@
 
 public class XDrive {
     MotorGroup motors;
-    DcMotor front;
-    DcMotor back;
-    DcMotor left;
-    DcMotor right;
+    DcMotor frontLeft;
+    DcMotor frontRight;
+    DcMotor backLeft;
+    DcMotor backRight;
     float f;
     float r;
     ImuEx imu;
 
     public void init(XDriveConfig config){
-        front = config.getFront();
-        back = config.getBack();
-        left = config.getLeft();
-        right = config.getRight();
+        frontLeft = config.getFrontLeft();
+        frontRight = config.getFrontRight();
+        backLeft = config.getBackLeft();
+        backRight = config.getBackRight();
 
-        front.setDirection(DcMotorSimple.Direction.FORWARD);
-        back.setDirection(DcMotorSimple.Direction.REVERSE);
-        left.setDirection(DcMotorSimple.Direction.REVERSE);
-        right.setDirection(DcMotorSimple.Direction.FORWARD);
+        frontLeft.setDirection(DcMotorSimple.Direction.FORWARD);
```

Comment:
for mecanum drive either left side or right side need to be reversed.  you have forward and reverse on each side